### DISCUSSION TITLE
storageclass per bucket

### DISCUSSION THREAD

**Discussion by andrewdunndev** [minio/minio#15774] [Q&A] [answered]
Is it possible to have a single instance and create two buckets with different storage classes?

I have eight drives, I want to run EC:4 by default, but for some of my less important files I'd prefer to run EC:2. Wondering if I can change the `storageclass` at the bucket level or if it has to be specified someplace else?

I guess the overarching question is, can a minio instance operate with more than one storageclass?

**Comment by harshavardhana** [comment, marked as answer]
Technically we can support it, right now this is not a priority.

**Reply by andrewdunndev** [reply]
So it doesn't function this way right now (as a per bucket setting). Is the only way to use reduced redundancy to pass a header in when writing data to the bucket?

**Reply by harshavardhana** [reply]
yes @andrewgdunn
